src/index.js

```diff
@@ -52,19 +52,24 @@ function buildInitAction() {
     return initFromLocation(rootLocation, path, query, hash);
 }
 
-initIconLibrary();
-simpleNumberLocalizer();
-dateFnsLocalizer();
-registerSpoilerElement();
-ReactDOM.render(
-    <Provider store={store}>
-        <App />
-    </Provider>,
-    document.getElementById("app-root")
-);
-store.dispatch(buildInitAction());
+if (document.contentType === 'text/plain') {
+    initIconLibrary();
+    simpleNumberLocalizer();
+    dateFnsLocalizer();
+    registerSpoilerElement();
+    ReactDOM.render(
+        <Provider store={store}>
+            <App/>
+        </Provider>,
+        document.getElementById("app-root")
+    );
+    store.dispatch(buildInitAction());
 
-// If you want your app to work offline and load faster, you can change
-// unregister() to register() below. Note this comes with some pitfalls.
-// Learn more about service workers: http://bit.ly/CRA-PWA
-serviceWorker.unregister();
+    // If you want your app to work offline and load faster, you can change
+    // unregister() to register() below. Note this comes with some pitfalls.
+    // Learn more about service workers: http://bit.ly/CRA-PWA
+    serviceWorker.unregister();
+} else {
+    document.body.innerText = `Page with content type ${document.contentType} is not supported for security reasons.
+    Let node owner know that he needs to upgrade his node to version >=0.9.0.`
```

Comment:
I would reword it slightly in more official tone:

Pages with content type '${document.contentType}' are not supported anymore for security reasons.
Please turn to administrator to upgrade the node software to version 0.9.0 or later.